internal/pkg/transformation/save_test.go

```diff
@@ -1,58 +1,49 @@
 package transformation
 
 import (
-	"os"
-	"path/filepath"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
 	"github.com/keboola/keboola-as-code/internal/pkg/json"
 	"github.com/keboola/keboola-as-code/internal/pkg/model"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils"
 )
 
 func TestSaveTransformationEmpty(t *testing.T) {
-	projectDir := t.TempDir()
-	blocksDir := filepath.Join(projectDir, `branch`, `config`, `blocks`)
-	assert.NoError(t, os.MkdirAll(blocksDir, 0755))
-
-	// Prepare
-	logger, _ := utils.NewDebugLogger()
-	record, source := createTransTestStructs("keboola.snowflake-transformation")
+	logger, fs, state, objectFiles := createTestFixtures(t)
+	blocksDir := filesystem.Join(`branch`, `config`, `blocks`)
+	assert.NoError(t, fs.Mkdir(blocksDir))
 
 	// Save
-	configContent, err := SaveBlocks(projectDir, logger, model.DefaultNaming(), record, source)
-	assert.NoError(t, err)
-	configContentJson, err := json.Encode(configContent, false)
+	err := GenerateBlockFiles(logger, fs, model.DefaultNaming(), state, objectFiles)
 	assert.NoError(t, err)
-	assert.Equal(t, `{}`, string(configContentJson))
+	configContent := json.MustEncodeString(objectFiles.Configuration.Content, false)
+	assert.Equal(t, `{}`, configContent)
 }
 
 func TestSaveTransformation(t *testing.T) {
-	projectDir := t.TempDir()
-	blocksDir := filepath.Join(projectDir, `branch`, `config`, `blocks`)
-	assert.NoError(t, os.MkdirAll(blocksDir, 0755))
+	logger, fs, state, objectFiles := createTestFixtures(t)
+	blocksDir := filesystem.Join(`branch`, `config`, `blocks`)
+	assert.NoError(t, fs.Mkdir(blocksDir))
 
 	// Prepare
-	record, source := createTransTestStructs("keboola.snowflake-transformation")
-	logger, logs := utils.NewDebugLogger()
-	source.Content.Set(`foo`, `bar`)
+	objectFiles.Configuration.Content.Set(`foo`, `bar`)
 	parameters := *utils.NewOrderedMap()
 	parameters.Set(`blocks`, []map[string]interface{}{
 		{
 			"name": "block1",
 			"codes": []map[string]interface{}{
 				{
 					"name": "code1",
-					"scripts": []string{
+					"script": []string{
```

Comment:
Tu bola chyba! v testoch, spravne sa kluc vola `script`, 
Kedze sa obsah suboru v teste nekontroloval, tak sa na to neprislo.

V kode aj v integracnych testoch to bolo vzdy spravne.

V testoch sa uz teraz testuje aj obsah (nie je to v tomto commite, lebo zaroven sa tie riadky menili na FS):
https://github.com/keboola/keboola-as-code/blob/c1944bd24a6e4fcbd2f4c68d260869c9746de8b1/internal/pkg/transformation/save_test.go#L77